### DISCUSSION TITLE
Question on chosing foundation_model when fine-tuning

### DISCUSSION THREAD

**Discussion by zjucm** [ACEsuit/mace#686] [Q&A] [answered]
Dear author and friends:
I am confused about the differences between setting foundation_model = "small" , "medium" , "large".
What do they mean?
Are they all trained on MP dataset?
And I want to know if I want to fine-tuning the model https://github.com/ACEsuit/mace-mp/releases/tag/mace_mp_0/MACE_MPtrj_2022.9.model ，what should I do to the "foundation_model"?
Thank you!

This is my job.sh:
#!/bin/bash
#SBATCH --job-name=cm
#SBATCH --nodes=1
#SBATCH --gres=gpu:2
#SBATCH -e err
#SBATCH --ntasks=16
source ~/.bashrc
conda activate mace
python test.py \
  --name="MACE" \
  --foundation_model="./MACE_MPtrj_2022.9.model" \
  --energy_key="REF_energy" \
  --virials_key="REF_virials" \
  --forces_key="REF_forces" \
  --train_file="train.xyz" \
  --valid_fraction=0.05 \
  --test_file="test.xyz" \
  --energy_weight=1.0 \
  --forces_weight=1.0 \
  --E0s="average" \
  --lr=0.001 \
  --scaling="rms_forces_scaling" \
  --batch_size=2 \
  --max_num_epochs=20 \
  --ema \
  --ema_decay=0.99 \
  --amsgrad \
  --default_dtype="float64" \
  --device=cuda \
  --seed=3 

 Is it correct?

**Comment by ilyes319** [comment, marked as answer]
Hello, the differences are described in the paper: https://arxiv.org/abs/2401.00096.
You can pass `--foundation_model="large"` and it will work too.
If you use that model, please select `--multiheads_finetuning=False`.

**Reply by zjucm** [reply]
Thank you！
